### ISSUE TITLE
Should be able to update relationships without changing the models

### ISSUE THREAD

**Issue by zachfedor** [samselikoff/ember-cli-mirage#615]
When using a `PUT` or `PATCH` route, I am not able to simply update relationships. Take the use case of a blog post. Updating the content of the post works as normal. In ember, it would look like:

```
let post = this.store.peekRecord('post', 1);
post.set('content', "Hello, World!");
post.save();
```

This sends a `PATCH` request to Mirage which calls the `update` method and everything is persisted. But let's say we want to remove a relationship and replace it with another. We aren't changing the model data, just updating relationships:

```
let currentAuthor = this.store.peekRecord('author', 1);
let newAuthor = this.store.peekRecord('author', 2);
let post = this.store.peekRecord('post', 1);
post.set('author', newAuthor);
post.save();
```

This also sends a `PATCH` request to Mirage which calls the `update` method but nothing is persisted. This is because the `_getAttrsForRequest` method is enforcing that the `model.attributes` has changed.

Notes:
- [_getAttrsForRequest should normalize relationships](https://github.com/samselikoff/ember-cli-mirage/issues/561)
- [patch routes](https://github.com/samselikoff/ember-cli-mirage/issues/612)
- [similar PR, but abandoned?](https://github.com/samselikoff/ember-cli-mirage/pull/572)
- [my PR](https://github.com/samselikoff/ember-cli-mirage/pull/614)

**Comment by samselikoff** [comment]
You should be able to do this with your own route handler, so I expect you mean the shorthand should take care of this automatically, right? I agree it should by the way.

**Comment by zachfedor** [comment]
actually, i think we resolved this in PR #614. i think i just forgot to close this issue out. sorry about that
